resources/geocoding/en/52.txt

```diff
@@ -0,0 +1,418 @@
+# Copyright (C) 2017 Pindrop Security, Inc.
+
+# Licensed under the Google Software Grant and Corporate Contributor License Agreement (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# https://cla.developers.google.com/clas/new?domain=DOMAIN_GOOGLE&kind=KIND_CORPORATE
+
+# Generated from:
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200008A0002PDFS.pdf
+# Internal statistics data (2015-07-10).
```

Comment:
You need to remove this "internal statistics data" line then, based on the conversation you're having, where you say it comes from ITU.